content/chat/rooms/moderation/index.textile

```diff
@@ -9,4 +9,24 @@ languages:
   - kotlin
 ---
 
-Moderation content to go here.
+Moderation is a crucial feature for chat rooms and online communities to maintain a safe, respectful, and engaging environment for all participants. Moderators help enforce community guidelines and remove potentially harmful content that can drive users away from an online experience.
+
+Moderation strategies can take many forms. Human moderators can sit and participate in the chat room, using community guidelines to make judgements on the chat content, taking action such as deleting a message when it is found to be in violation of standards. Many modern approaches involve moderation engines and Artificial Intelligence models, which can screen content in order to filter out harmful messages before they are allowed into the chat room, without the need for human moderators. Many of these are highly configurable, allowing you to screen content across multiple categories to suit your needs. Hybrid approaches can make the best of both worlds, employing AI to pre-screen messages, with human moderators able to make judgement calls on edge-cases or in response to user feedback.
+
+Ably Chat supports a variety of moderation options in chat rooms, to help you keep your participants safe and engaged.
+
+h2(#types). Types of moderation
+
+Moderation with Ably falls into two flavours: before and after publish.
```

Comment:
```suggestion
Moderation with Ably falls into two ways: before and after publish.
```

`Flavours` feels somewhat informal to me. What do you think?

Reply:
Diluted the flavour